### PR TITLE
reflect the workflow change in contrib guide

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 9418bd6b is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit a1cf4247 is formatted properly.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -215,10 +214,10 @@ Follow the common rules for making easily mergable and reviewable PRs:
...
 
 The workflow is based on [GitLab
-flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html), i.e., a `master`
-branch with `feature` branches being merged into the `master` branch. Depending
-on your access rights, you may open the `feature` branch in this repository, on
-in your fork.
+flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html), i.e., a `develop`
+branch with `feature` branches being merged into the `develop` branch, all
+periodically merged to `master` branch. Depending on your access rights, you
+may open the `feature` branch in this repository, on in your fork.
```
Comment (laurentheirendt) [inline .github/CONTRIBUTING.md:220]: `or`. That would actually be the preferred way.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -169,8 +169,7 @@ submitting the pull requests.
 2. Make a pull request where you describe the motivation and expected outcome
    for the users. Specifically, consider any possible incompatibilities, and the
    necessity to increment the version number after your changes are applied.
-   Label your pull request with a [semantic emoji](https://gitmoji.dev/) in the
```
Comment (laurentheirendt) [inline .github/CONTRIBUTING.md:172]: 👍🏼

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 04abfd7a is formatted properly.

**Comment by exaexa** [comment]
The comment from the review is fixed -> merging

**Pull request merged**